pkg/object/object.go

```diff
@@ -70,10 +75,30 @@ type Volume struct {
 	StorageClass *string
 }
 
+type Secret struct {
+	Name string
+	Data []SecretData
+}
+
+type SecretData struct {
+	Key       string
+	Plaintext *string
+	Base64    *string
+	File      *string
+}
+
 type OpenCompose struct {
 	Version  string
 	Services []Service
 	Volumes  []Volume
+	Secrets  []Secret
+}
+
+type Input struct {
```

Comment:
hmm :-( It really can't be in another place? It would be nice to keep `object` clean and have only stuff there only stuff related to spec.